README.md

```diff
@@ -148,6 +148,62 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 }
 ```
 
+### Bluetooth Low-energy (BLE) Example
+
+This example requires a powered and flashed Meshtastic radio with BLE enabled. You need to pair it first using your operating system utilities. PIN might be needed.
+
+> [!NOTE]
+> You need `bluetooth-le` feature enabled for Bluetooth low energy support.
+
+```rust
+/// This example connects via Bluetooth LE to the radio, and prints out all received packets.
+extern crate meshtastic;
+
+use std::io::{self, BufRead};
+use std::time::Duration;
+
+use meshtastic::api::StreamApi;
+use meshtastic::utils;
+use meshtastic::utils::stream::BleId;
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let stream_api = StreamApi::new();
+
+    println!("Enter the short name of a BLE device to connect to:");
+
+    let stdin = io::stdin();
+    let entered_name = stdin
+        .lock()
+        .lines()
+        .next()
+        .expect("Failed to find next line")
+        .expect("Could not read next line");
+
+    // You can also use `BleId::from_mac_address("AB")` instead of `BleId::from_name()`.
+    let ble_stream =
+        utils::stream::build_ble_stream(&BleId::from_name(&entered_name), Duration::from_secs(5))
+            .await?;
+    let (mut decoded_listener, stream_api) = stream_api.connect(ble_stream).await;
+
+    let config_id = utils::generate_rand_id();
+    let stream_api = stream_api.configure(config_id).await?;
+
+    // This loop can be broken with ctrl+c, by disabling bluetooth or by turning off the radio.
+    while let Some(decoded) = decoded_listener.recv().await {
+        println!("Received: {:?}", decoded);
+    }
+
+    // Note that in this specific example, this will only be called when
+    // the radio is disconnected, as the above loop will never exit.
+    // Typically you would allow the user to manually kill the loop,
+    // for example with tokio::select!.
+    let _stream_api = stream_api.disconnect().await?;
+
+    Ok(())
+}
+```
+
```

Comment:
By no means a blocker, and i recognize you are just following along with the existing patterns in the README here, but i find that example code duplicated  in markdown files usually drift from the real `examples/` over time.

In a separate PR, itmight be nice to link to the examples themselves from the README, with descriptions of each, but omitting their contents to avoid this, and to also ensure the code continues to compile and be correctly updated in the future.

Reply:
I agree, we can clean this up later. However, we can't directly include them, we'll need to provide links: https://stackoverflow.com/a/35080768